python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -40,6 +40,52 @@ def test_init_via_list_of_empty_tuples(rows):
     assert_eq(pdf, gdf, check_like=True)
 
 
+@pytest.mark.parametrize(
+    "dict_of_series",
+    [
+        {"a": pd.Series([1.0, 2.0, 3.0])},
+        {"a": pd.Series([1.0, 2.0, 3.0], index=[4, 5, 6])},
+        {
+            "a": pd.Series([1.0, 2.0, 3.0], index=[4, 5, 6]),
+            "b": pd.Series([1.0, 2.0, 4.0], index=[1, 2, 3]),
+        },
+        {"a": [1, 2, 3], "b": pd.Series([1.0, 2.0, 3.0], index=[4, 5, 6])},
+    ],
```

Comment:
Would add a couple of cases where there's two series with duplicate values in the index both to check that we properly catch and error if they differ, and that we succeed if they're the same.

Reply:
Done